store/tikv/mocktikv/cop_handler_dag.go

```diff
@@ -95,7 +95,11 @@ func (h *rpcHandler) handleCopDAGRequest(req *coprocessor.Request) *coprocessor.
 		chunks = appendRow(chunks, data, rowCnt)
 		rowCnt++
 	}
-	return buildResp(chunks, err)
+	counts := make([]int64, len(dagReq.Executors))
+	for offset := len(dagReq.Executors) - 1; e != nil; e, offset = e.GetSrcExec(), offset-1 {
```

Comment:
How about just
```go
for i, exec := range dagReq.Executors {
    counts[i] = exec.Count() 
}
```

Reply:
We cannot do this, the `dag.Executors` is just protobuf structure.